recipes/mcroni/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set version = "v1.0.0" %}
```

The version must not start with a "v", see also https://github.com/bioconda/bioconda-recipes/issues/19456
```suggestion
{% set version = "1.0.0" %}
```